doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,658 @@
+   {
+       (dependencyobject as PrismJsView)?.UpdateDisplay(args.OldValue as string, args.NewValue as string);
+   }
+   ```
+
+4. We also need to update the result when the control is loaded in the DOM. So we need to change the constructor again like this:
+
+   ``` csharp
+   public PrismJsView() : base("code") // PrismJS requires a <code> element
+   {
+       // This is required to set to <code> style for PrismJS to works well
+       // https://github.com/PrismJS/prism/issues/1237#issuecomment-369846817
+       SetStyle("white-space", "pre-wrap");
+   
+       // Update the display when the element is loaded in the DOM
+       Loaded += (snd, evt) => UpdateDisplay(newLanguage: Language);
+   }
+   ```
+
+5. Compile & run.  It should work like this:
+   ![image-20200415135422628](image-20200415135422628.png)
+
+## 🔬 Going further
+
+This sample is a very simple integration as there is no _callback_ from HTML to managed code and _PrismJS_ is a self-contained framework (it does not download any other javascript dependencies).
+
+Some additional improvements can be done to make the code more production ready:
+
+* **Make the control multi-platform**. A simple way would be to use a WebView on other platforms, giving the exact same text-rendering framework everywhere. The code of this sample won't compile on other targets.
+* **Create script files instead of generating dynamic JavaScript**. That would have the advantage of improving performance and make it easier to debug the code. A few projects are also using TypeScript to generate JavaScript. This approach is done by Uno itself for the `Uno.UI.Wasm` project: https://github.com/unoplatform/uno/tree/master/src/Uno.UI.Wasm.
+* **Support more PrismJS features**. There are many [_plugins_ for PrismJS](https://prismjs.com/#plugins) that can be used. Most of them are very easy to implement.
+
+
+
+# Article 2 - Callback to app from JavaScript
+
+In the previous article, a simple _syntax highlighter_ was used to enhance the display of text in HTML. But it is not enough for most apps: it's often required for JavaScript components to call back into the C# side of application. The easiest way to do that in Uno for WebAssembly is by using [_DOM Events_](https://developer.mozilla.org/docs/Web/Guide/Events/Creating_and_triggering_events). Applications using Uno can [consume DOM Events and CustomEvents](https://platform.uno/docs/articles/wasm-custom-events.html) very easily.
+
+Let's create an application illustrating how to use this feature.
+
+# Sample 2 - Integration of Flatpickr
+
+📝 [Flatpickr](https://flatpickr.js.org/) is a lightweight, self-contained date and time picker. It's an easy way to explore how a JavaScript can call back to the managed application code. In this case, this will be used to report when the picker is opened and a date and time was picked.
+
+## 1. Create the solution in VisualStudio
+
+📝 This part is very short because it is similar to previous article:
```

Comment:
```suggestion
📝 This part is very short because it is similar to previous article:
```
```suggestion
📝 This part is very short because it is similar to the previous article:
```